src/components/EntitiesCount/EntitiesCount.tsx

```diff
@@ -0,0 +1,37 @@
+import {Label} from '@gravity-ui/uikit';
+
+interface EntitiesCountProps {
+    countTotal: number | string;
+    countCurrent?: number | string;
```

Comment:
seems to me that total should be optional, not current
the current counter is always available thru the collection size, while the total can be unknown

Reply:
even in the current use case code can be simplified if you swap the optionals